From 937db1345bd4ba9a973c9a726c129b6a2aa8929e Mon Sep 17 00:00:00 2001
From: Jernej Skrabec <jernej.skrabec@gmail.com>
Date: Sun, 9 May 2021 12:00:46 +0200
Subject: [PATCH 240/486] ARM: dts: sun8i: v40: bananapi-m2-berry: Enable HDMI
 audio

BananaPi M2 Berry has HDMI output and thus supports HDMI audio. Enable
it.

Signed-off-by: Jernej Skrabec <jernej.skrabec@gmail.com>
---
 arch/arm/boot/dts/sun8i-v40-bananapi-m2-berry.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/sun8i-v40-bananapi-m2-berry.dts b/arch/arm/boot/dts/sun8i-v40-bananapi-m2-berry.dts
index 434871040..a661b1a60 100644
--- a/arch/arm/boot/dts/sun8i-v40-bananapi-m2-berry.dts
+++ b/arch/arm/boot/dts/sun8i-v40-bananapi-m2-berry.dts
@@ -160,6 +160,10 @@ axp22x: pmic@34 {
 
 #include "axp22x.dtsi"
 
+&i2s2 {
+	status = "okay";
+};
+
 &mmc0 {
 	vmmc-supply = <&reg_dcdc1>;
 	bus-width = <4>;
@@ -275,6 +279,10 @@ &reg_eldo3 {
 	regulator-name = "vdd1v2-sata";
 };
 
+&sound_hdmi {
+	status = "okay";
+};
+
 &tcon_tv0 {
 	status = "okay";
 };
-- 
2.35.3

